package.json

```diff
@@ -14,6 +16,7 @@
     "eject": "react-scripts eject"
   },
   "devDependencies": {
+    "eslint-plugin-react-hooks": "^4.6.0",
```

Comment:
Nice! I don't think we covered this? But its a great idea to follow these linting rules!